npm/vue2/src/index.ts

```diff
@@ -318,16 +318,18 @@ function getComponentDisplayName (componentOptions: any): string {
 
 /**
  * Mounts a Vue component inside Cypress browser.
- * @param {object} component imported from Vue file
+ * @param {VueComponent} component imported from Vue file
```

Comment:
```suggestion
 * @param {VueComponent} component imported from Vue file
 * @param {MountOptionsArgument} optionsOrProps used to pass options to component being mounted
 * @returns {Cypress.Chainable<{wrapper: Wrapper<T>, component: T}
```
Documenting second param and return type